src/date.c

```diff
@@ -1566,48 +1566,7 @@ static void ctimestampFunc(
 ** as the user-data for the function.
 */
 
-#ifdef FREEBSD_KERNEL
-
-//todo: STELIOS
-static void currentTimeFunc(
-  sqlite3_context *context,
-  int argc,
-  sqlite3_value **argv
-){
-  // time_t t;
-  //char *zFormat = (char *)sqlite3_user_data(context);
-  // sqlite3_int64 iT;
-  // struct tm *pTm;
-  // struct tm sNow;
-  char zBuf[20];
-
-  /*
-  UNUSED_PARAMETER(argc);
-  UNUSED_PARAMETER(argv);
-
-  iT = sqlite3StmtCurrentTime(context);
-  if( iT<=0 ) return;
-  t = iT/1000 - 10000*(sqlite3_int64)21086676;
-#if HAVE_GMTIME_R
-  pTm = gmtime_r(&t, &sNow);
-#else
-  sqlite3_mutex_enter(sqlite3MutexAlloc(SQLITE_MUTEX_STATIC_MAIN));
-  pTm = gmtime(&t);
-  if( pTm ) memcpy(&sNow, pTm, sizeof(sNow));
-  sqlite3_mutex_leave(sqlite3MutexAlloc(SQLITE_MUTEX_STATIC_MAIN));
-#endif
-  if( pTm ){
-    strftime(zBuf, 20, zFormat, &sNow);
-    sqlite3_result_text(context, zBuf, -1, SQLITE_TRANSIENT);
-  }
-  */
-  printf("Warning: currentTimeFunc - function is not implemented on FreeBSD Kernel. This code should not be executed!\n");
-  zBuf[0] = '0';
-  for (int i = 1; i < 20; i++) zBuf[i] = zBuf[i-1];
-  sqlite3_result_text(context, zBuf, -1, SQLITE_TRANSIENT);
-}
-
-#elif defined(LINUX_KERNEL_BUILD)
+#ifdefined(LINUX_KERNEL_BUILD)
```

Comment:
```suggestion
#if defined(LINUX_KERNEL_BUILD)
```
I'd separate the #if from the defined to keep it consistent with the rest of the code, but feel free to ignore this comment